### PR TITLE
Removed custom_object.py and created a new file namespaced_custom_object.py which is more specific to usage of namespaced custom resource 

### LINKED ISSUE #1438: Improve the custom_object.py to be more specific to usage of namespaced custom resource
**Link to the issue (please include a link to the specific documentation or example)**:
https://github.com/kubernetes-client/python/tree/master/examples

**Description of the issue (please include outputs or screenshots if possible)**:

The [examples/custom_object.py](https://github.com/kubernetes-client/python/blob/master/examples/custom_object.py) example script can be improved by making it more specific to usage of  [kubernetes.client.CustomObjectsApi](https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/CustomObjectsApi.md) for a namespaced scope custom resource.

Following are the suggestion for improvement:
- rename the file to `namespaced_custom_object.py`
- include example usage of `patch_namespaced_custom_object` method

### PR DESCRIPTION
**Pull request by Yashks1994**
#### What type of PR is this?
/kind cleanup
/kind documentation

#### What this PR does / why we need it:
Removed the custom_object.py and created a new file as namespaced_custom_object.py which is more specific to usage of namespaced custom resource.

#### Which issue(s) this PR fixes:

Fixes #1438

### CODE REVIEW

File: examples/namespaced_custom_object.py [hunk 1]
```diff
@@ -16,9 +16,7 @@
 Uses a Custom Resource Definition (CRD) to create a custom object, in this case
 a CronTab. This example use an example CRD from this tutorial:
 https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/
-
 The following yaml manifest has to be applied first:
-
```
Comment (roycaihw) [inline examples/namespaced_custom_object.py:21]: let's keep these empty lines. Otherwise LGTM

Comment (Yashks1994) [reply examples/namespaced_custom_object.py]: Done

**Comment by Priyankasaggu11929** [comment]
@Yashks1994, just a note.

Git takes renaming of the  file with name `custom_object.py` to `namespaced_custom_object.py` as two steps:
    - deleting the file `custom_object.py`
    - and creating a new file `namespaced_custom_object.py` having the same contents as above.

To reflect the deletion of file `custom_object.py`, you need to add the deleted file as well in your git staging & then commit both the deleted file & new file.

Doing that, in the `Files changed` section of this PR, both the deleted file (custom_object.py) & new file (namespaced_object.py) will reflect.

**Comment by Yashks1994** [comment]
@roycaihw needs your /lgtm as CI build passed.

**Comment by roycaihw** [comment]
> The file renaming would look something like this in the PR's Files Changed section.

@Priyankasaggu11929 I was surprised when I saw that the first time. I didn't expect Github to render it that way but it's really beautiful. 

@Yashks1994 LGTM. Please squash the commits as I suggested in https://github.com/kubernetes-client/python/pull/1439#issuecomment-828926719: 
- one commit for adding patch example to custom_object.py
- one commit for renaming the file to namespaced_custom_object.py

File: examples/namespaced_custom_object.py [hunk 2]
```diff
@@ -99,30 +98,28 @@ def main():
...
     )
-    print("Resource deleted")
-
+    print("Resource details:")
+    pprint(patch_resource)
 
-    # patch the namespaced custom object to update the `spec.cronSpec` field
-    patch_resource = api.patch_namespaced_custom_object(
+    # delete it
+    api.delete_namespaced_custom_object(
```
Comment (roycaihw) [inline examples/namespaced_custom_object.py:114]: why do we have a commit that reverse the order of `patch_namespaced_custom_object` and `delete_namespaced_custom_object`?

Comment (roycaihw) [reply examples/namespaced_custom_object.py:114]: https://github.com/kubernetes-client/python/pull/1444/commits is what I was asking for, if that makes sense

Comment (Yashks1994) [reply examples/namespaced_custom_object.py:114]: I kept the order like this,
we have create, get, patch, delete

Comment (roycaihw) [reply examples/namespaced_custom_object.py:114]: currently the two PR have the same code change, but the commits are different

Commits should represent meaningful milestones or units of work. Currently the commits in this PR are: 
- your first commit renames the file and adds a patch method after the delete method, which means the example has a bug and will run into an error
- your second commit reverses the order of the patch method and the delete method to fix the bug in the first commit

please take a look at https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md#squash-commits. The second commit is a "Work in progress" (it fixes a bug in the first commit during development), so please squash it. 

again, I think having two commits like the following will be most reasonable and readable:
- one commit for adding patch example to custom_object.py
- one commit for renaming the file to namespaced_custom_object.py

at this point I'm also okay with having just one commit, which will be more reasonable than the current two commits in this PR

Comment (Yashks1994) [reply examples/namespaced_custom_object.py:114]: I am keeping the one commit and with proper commit message.

**Comment by roycaihw** [comment]
Thanks! 
/lgtm

**Pull request merged**